demo/laneLoader.js

```diff
@@ -528,143 +533,177 @@ function updateSegments(laneSegments, clonedBoxes, prevIsContains, point, index,
+  for (let ii = 0, len = laneGeometries.leftAnomalies.length; ii < len; ii++) {
+    const point = laneGeometries.leftAnomalies[ii];
+    const aAnomaly = new Potree.Annotation({
+      title: 'L ' + [ii + 1],
+      position: point,
+      cameraPosition: new THREE.Vector3(point.x, point.y, point.z + 20),
+      cameraTarget: point
+    });
+    aLeft.add(aAnomaly);
+  }
+  for (let ii = 0, len = laneGeometries.rightAnomalies.length; ii < len; ii++) {
+    const point = laneGeometries.rightAnomalies[ii];
+    const aAnomaly = new Potree.Annotation({
+      title: 'R ' + [ii + 1],
+      position: point,
+      cameraPosition: new THREE.Vector3(point.x, point.y, point.z + 20),
+      cameraTarget: point
+    });
+    aRight.add(aAnomaly);
+  }
+}
+
+
+// Adds lane geometries to viewer
+function addLaneGeometries (laneGeometries, lanesLayer) {
+  for (let ii = 0, len = laneGeometries.all.length; ii < len; ii++) {
+    lanesLayer.add(laneGeometries.all[ii]);
+  }
+  viewer.scene.scene.add(lanesLayer);
+
+  // add lane anomaly geometries
+  if (laneGeometries.leftAnomalies.length !== 0 ||
+    laneGeometries.rightAnomalies.length !== 0) {
+    addAnnotations(laneGeometries);
+  }
+}
+
 
+// Load Lanes Truth Data:
+export async function loadLanesCallback(s3, bucket, name, callback) {
+  let filename, tmpSupplierNum;
+  tmpSupplierNum = -1;
+  await loadLanes(s3, bucket, name, filename, tmpSupplierNum, window.annotateLanesModeActive, viewer.scene.volumes, (laneGeometries) => {
+    // need to have Annoted Lanes layer, so that can have original and edited lanes layers
+    let lanesLayer = new THREE.Group();
+    lanesLayer.name = "Lanes";
+    addLaneGeometries(laneGeometries, lanesLayer);
+    viewer.scene.dispatchEvent({
+      "type": "truth_layer_added",
+      "truthLayer": lanesLayer
+    });
+    if (callback) {
+      callback();
+    }
+  });
+
+  if (visualizationMode == "aptivLanes") {
+    async function loadLanesHelper(layerName, filename, s) {
+      try {
+        await loadLanes(s3, bucket, name, filename, s, window.annotateLanesModeActive, viewer.scene.volumes, (laneGeometries) => {
+          let lanesLayer = new THREE.Group();
+          lanesLayer.name = layerName;
+          addLaneGeometries(laneGeometries, lanesLayer);
+          viewer.scene.dispatchEvent({
+            "type": "map_provider_layer_added",
+            "mapLayer": lanesLayer
+          });
+        });
+      } catch (e) {
+        console.log(`Couldn't load ${filename}: ${e}`);
+      }
+    };
+    const laneNum = [1, 2, 3];
+    const laneDirection = ["EB", "WB"];
+    const supplierNum = [1, 2, 3];
+    let filename, layerName, datasetName;
+
+    for (let s of supplierNum) {
+      for (let d of laneDirection) {
+        for (let n of laneNum) {
+          layerName = `Supplier${s}_${d}_Lane${n}`;
+          filename = `Supplier${s}_${d}_Lane${n}.fb`;
+          loadLanesHelper(layerName, filename, s);
+        }
+      }
+    }
+    layerName = `TomTom`;
```

Comment:
If strict mode doesn't let you declare these here (`const layerName = ...`) then I'd just skip the names, and do:
```
    loadLanesHelper(`TomTom`, `I-75-North_potree.fb`, 1);
```